src/common/commandExecutor.ts

```diff
@@ -120,8 +122,8 @@ export class CommandExecutor {
      * Executes a react native command and waits for its completion.
      */
     public spawnReactCommand(command: string, args?: string[], options: Options = {}): ISpawnResult {
-        const reactCommand = HostPlatform.getNpmCliCommand(CommandExecutor.ReactNativeCommand);
-        return this.spawnChildProcess(reactCommand, this.combineArguments(command, args), options);
+        const reactCommand = CommandExecutor.ReactNativeCLI;
+        return this.spawnChildProcess("node", [reactCommand, command, ...args], options);
```

Comment:
Packager, you meant. Anyway, this call is effectively has the same result as `this.spawnChildProcess(reactCommand, this.combineArguments(command, args), options)`  so I wonder if the change was really required.